### DISCUSSION TITLE
Building Free-Threaded Python 3.13.0a1 in PowerShell

### DISCUSSION THREAD

**Discussion by anzhi0708** [pyenv/pyenv#2986] [Q&A] [answered]
I've been using macOS/Linux for many years and I'm not familiar with Powershell/Windows. I tried
```powershell
$Env:PYTHON_CONFIGURE_OPTS='--disable-gil'
pyenv install 3.13.0a1
```
and
```powershell
$PYTHON_CONFIGURE_OPTS='--disable-gil'; pyenv install 3.13.0a1
```
but neither worked. Any advice? I'll be very grateful :3

**Comment by native-api** [comment, marked as answer]
[Pyenv doesn't support Windows outside of WSL](https://github.com/pyenv/pyenv?tab=readme-ov-file#windows). You must be using [Pyenv-Win](https://github.com/pyenv-win/pyenv-win). Ask there.
